examples/chip-tool/commands/icd/ICDCommand.cpp

```diff
@@ -64,13 +65,72 @@ CHIP_ERROR ICDListCommand::RunCommand()
     return CHIP_NO_ERROR;
 }
 
+CHIP_ERROR ICDWaitForDeviceCommand::RunCommand()
+{
+    if (!IsPeerLIT())
+    {
+        ChipLogError(chipTool, "The device is not a registered LIT-ICD device.");
+        return CHIP_ERROR_NOT_FOUND;
+    }
+    mInterestedNode = ScopedNodeId(GetDestinationId(), CurrentCommissioner().GetFabricIndex());
+    ChipLogError(chipTool, "Please trigger the device active mode.");
+    return CHIP_NO_ERROR;
+}
+
+void ICDWaitForDeviceCommand::OnCheckInComplete(const chip::app::ICDClientInfo & clientInfo)
+{
+    DefaultCheckInDelegate::OnCheckInComplete(clientInfo);
+
+    if (clientInfo.peer_node != mInterestedNode)
+    {
+        ChipLogDetail(chipTool, "The node " ChipLogFormatScopedNodeId " is not the one we are interested in.",
+                      ChipLogValueScopedNodeId(clientInfo.peer_node));
+        return;
+    }
+
+    ChipLogDetail(chipTool, "Received check-in message from the node, send stay active request to the device.");
+    mInterestedNode = ScopedNodeId();
+
+    CHIP_ERROR err = ClusterCommand::RunCommand();
```

Comment:
should it be sendCommand?

Reply:
`RunCommand` is intentionally used here.

Given the complexity of sending a command `RunCommand` does a lot and calls `SendCommand` finally.

Reply:
i see, thanks